### ISSUE TITLE
Server crashes when deep linking to an API route

### ISSUE THREAD

**Issue by lydemann** [analogjs/analog#363]
### Please provide the environment you discovered this bug in.

Just generate a new Analog app with Nx and type in /api/hello.

First time goes well and but if you refresh it fails.

### Which area/package is the issue in?

router

### Description

Just generate a new Analog app with Nx and type in /api/hello.

First time goes well and but if you refresh it fails with:

"RuntimeError: NG04002: Cannot match any routes. URL Segment: 'favicon.ico'"

Video:
https://www.loom.com/share/dd31d89a0f32418cb19073404a3756f5

### Please provide the exception or error you saw

```true
RuntimeError: NG04002: Cannot match any routes. URL Segment: 'favicon.ico'
```

### I would be willing to submit a PR to fix this issue

- [X] Yes
- [ ] No

**Comment by masoudtahmasebi93** [comment]
This can be fixed, by moving `favicon.ico` file from the src directory to the root directory and also changing the import in `index.html` file from:
`<link rel="icon" type="image/x-icon" href="/src/favicon.ico">`
to
`<link rel="icon" type="image/x-icon" href="/favicon.ico">`
I will try to raise a merge request to fix this. Maybe the community sees this and applies it.

**Comment by goetzrobin** [comment]
@masoudtahmasebi93 thanks for pointing that out!

**Comment by brandonroberts** [comment]
This have been fixed with the latest release in a971e957bfc1392842b085bccc292d3b09af8678
